deployments/qat_plugin/overlays/qat_initcontainer/qat.conf

```diff
@@ -1 +0,0 @@
-ServicesEnabled=sym;asym
```

Comment:
do you plan to add a new test case afterwards that services configuration works?

Reply:
Umm, it still works with a config, right?
We just want to remove 'default' so users can just put if they need to set all services as 'sym;asym' or 'dc'.
If config file is not present, then it just keeps as it is.

Reply:
> Umm, it still works with a config, right?

yes, we want to have a test case in e2e that if user has created the config, the provisioning works

Reply:
done!
e2e-qat fails, but it will be solved after #1395 is merged (and this should be rebased).!